src/main/resources/database/changes/release-10.49.0/add_notes_to_sample_summary.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE sample.samplesummary ADD COLUMN notes VARCHAR(250);
```

Comment:
Can we not just create the column as a text datatype first time around? if not it seems like both changes can live in one sql file as it's all related to adding notes column

Reply:
In my experience it is risky and difficult to revise liquibase scripts.  As I can't be sure where it has been deployed I'd have to get both hashes and add them explicitly which I'd rather not do. We might well be alright in this case but I have learned that is necessary when changing liquibase. Better safe than sorry

Reply:
Sorry I meant just merging add_notes_to_sample_summary.sql and alter_notes_column_to_text.sql into one file or just doing something like `ALTER TABLE sample.samplesummary ADD COLUMN notes TEXT;` and having the one line

Reply:
The change to a text field was made after the PR had been raised in response to an error spotted by Tej.  As I knew the code had been deployed elsewhere, even if it was most likely to a transient environment, it meant I would have to do the change properly i.e. with hashes or to add an additional script. I chose the latter as it was simpler.

Reply:
I don't think it should be a problem as it won't have got into any other environment other than peoples local transient environment but sure